--- Transceiver52M/UHDDevice.cpp.original	2019-12-29 01:59:17.508157052 +0000
+++ Transceiver52M/UHDDevice.cpp	2019-12-30 03:38:38.545831899 +0000
@@ -26,7 +26,7 @@
 #include <uhd/property_tree.hpp>
 #include <uhd/usrp/multi_usrp.hpp>
 #include <uhd/utils/thread_priority.hpp>
-#include <uhd/utils/msg.hpp>
+//#include <uhd/utils/log_add.hpp>
 
 #ifdef HAVE_CONFIG_H
 #include "config.h"
@@ -318,6 +318,7 @@
     since we already report using the logging facility. Direct
     everything else appropriately.
  */
+/*
 void uhd_msg_handler(uhd::msg::type_t type, const std::string &msg)
 {
 	switch (type) {
@@ -334,6 +335,8 @@
 		break;
 	}
 }
+*/
+
 
 uhd_device::uhd_device(int sps, bool skip_rx)
 	: tx_gain(0.0), tx_gain_min(0.0), tx_gain_max(0.0),
@@ -664,7 +667,7 @@
 	setPriority();
 
 	// Register msg handler
-	uhd::msg::register_handler(&uhd_msg_handler);
+	//uhd::msg::register_handler(&uhd_msg_handler);
 
 	// Start asynchronous event (underrun check) loop
 	async_event_thrd.start((void * (*)(void*))async_event_loop, (void*)this);
